Rules/Playbooks/Smugglers.md

```diff
@@ -0,0 +1,105 @@
+> Copy this playbook to create a playable crew, then link back to the original. Follow the directions in [[01 Crew creation]] to get started.
+
+## Crew Stats
+
+### reputation
+
+### lair
+
+### trackers
+
+| capital       | rep          | turf        | tier        | hold        | XP          |
+| ------------- | ------------ | ----------- | ----------- | ----------- | ----------- |
+| `clock 2 / 4` | `clock 0/12` | `clock 0/6` | `clock 0/5` | `clock 1/1` | `clock 0/9` |
```

Comment:
Slightly frustrating, there is an open [issue](https://github.com/tokenshift/obsidian-progress-clocks/issues/5) against the clocks plugin where interacting with inline clocks in tables doesn't update the source doc. I may try to fix that myself at some point, but the workaround is to edit the source text of the clock rather than just interacting with the UI.